diff --git a/internal/vendor/github.com/coreos/update-ssh-keys/src/authorized_keys_d/as_user/as_user.c b/internal/vendor/github.com/coreos/update-ssh-keys/src/authorized_keys_d/as_user/as_user.c
index abe79d9..178c070 100644
--- a/internal/vendor/github.com/coreos/update-ssh-keys/src/authorized_keys_d/as_user/as_user.c
+++ b/internal/vendor/github.com/coreos/update-ssh-keys/src/authorized_keys_d/as_user/as_user.c
@@ -22,6 +22,7 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <unistd.h>
+#include <sys/wait.h>
 
 #include "as_user.h"
 
